### ISSUE TITLE
Language construct with parentheses inside condition is not reported

### ISSUE THREAD

**Issue by morozov** [slevomat/coding-standard#1077]
```
$ composer show | grep -P 'coding-standard|php_codesniffer'

doctrine/coding-standard                       8.1.0     The Doctrine Coding Standard is ...
slevomat/coding-standard                       6.3.11    Slevomat Coding Standard for PHP...
squizlabs/php_codesniffer                      3.5.6     PHP_CodeSniffer tokenizes PHP, J...
```
Consider the following code example:

```php
<?php

include_once(__FILE__);

if (include_once(__FILE__)) {
    echo 'OK', PHP_EOL;
}
```
Only one of the two violations are reported:
```
$ phpcs --sniffs=SlevomatCodingStandard.ControlStructures.LanguageConstructWithParentheses test.php

E 1 / 1 (100%)

FILE: test.php
------------------------------------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
------------------------------------------------------------------------------------------------
 3 | ERROR | [x] Usage of language construct "include_once" with parentheses is disallowed.
   |       |     (SlevomatCodingStandard.ControlStructures.LanguageConstructWithParentheses.UsedWithParentheses)
------------------------------------------------------------------------------------------------
```

**Comment by kukulich** [comment]
Thank you for your report.

Fixed in https://github.com/slevomat/coding-standard/commit/1ab8f78768c2cd2acc2e0aacb31ac57df5f8a719
